pep-9999.rst

```diff
@@ -0,0 +1,342 @@
+PEP: 9999
+Title: Lazy Imports
+Author: Germán Méndez Bravo <german.mb@gmail.com>, Carl Meyer <carl.j.meyer@gmail.com>
+Sponsor: Barry Warsaw <barry@python.org>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 29-Apr-2022
+Python-Version: 3.12
+Discussions-To: https://discuss.python.org
```

Comment:
Please remove this and update the PEP with the link to the thread when you've opened it
```suggestion
```